src/init/pkg_installer.sh

```diff
@@ -2,52 +2,53 @@
+cp -p /etc/ossec-init.conf ${WAZUH_HOME}/tmp_bkp/etc
 
-tar czf ${DIRECTORY}/backup/backup_${VERSION}_[${BDATE}].tar.gz -C ${DIRECTORY}/tmp_bkp . >> ${DIRECTORY}/logs/upgrade.log 2>&1
-rm -rf ${DIRECTORY}/tmp_bkp
+tar czf ${WAZUH_HOME}/backup/backup_${WAZUH_VERSION}_[${BDATE}].tar.gz -C ${WAZUH_HOME}/tmp_bkp . >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
+rm -rf ${WAZUH_HOME}/tmp_bkp
 
 # Installing upgrade
-echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade started." >> ${DIRECTORY}/logs/upgrade.log
-chmod +x ${DIRECTORY}/var/upgrade/install.sh
-${DIRECTORY}/var/upgrade/install.sh >> ${DIRECTORY}/logs/upgrade.log 2>&1
+echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade started." >> ${WAZUH_HOME}/logs/upgrade.log
+chmod +x ${WAZUH_HOME}/var/upgrade/install.sh
+${WAZUH_HOME}/var/upgrade/install.sh >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
 
 # Check installation result
 RESULT=$?
-echo "$(date +"%Y/%m/%d %H:%M:%S") - Installation result = ${RESULT}" >> ${DIRECTORY}/logs/upgrade.log
+echo "$(date +"%Y/%m/%d %H:%M:%S") - Installation result = ${RESULT}" >> ${WAZUH_HOME}/logs/upgrade.log
 
 # Wait connection
 status="pending"
 COUNTER=30
 while [ "$status" != "connected" -a $COUNTER -gt 0  ]; do
-    . ${DIRECTORY}/var/run/wazuh-agentd.state >> ${DIRECTORY}/logs/upgrade.log 2>&1
+    . ${WAZUH_HOME}/var/run/wazuh-agentd.state >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
     sleep 1
     COUNTER=$[COUNTER - 1]
-    echo "$(date +"%Y/%m/%d %H:%M:%S") - Waiting connection... Status = "${status}". Remaining attempts: ${COUNTER}." >> ${DIRECTORY}/logs/upgrade.log
+    echo "$(date +"%Y/%m/%d %H:%M:%S") - Waiting connection... Status = "${status}". Remaining attempts: ${COUNTER}." >> ${WAZUH_HOME}/logs/upgrade.log
 done
 
 # Check connection
 if [ "$status" = "connected" -a $RESULT -eq 0  ]; then
-    echo "$(date +"%Y/%m/%d %H:%M:%S") - Connected to manager." >> ${DIRECTORY}/logs/upgrade.log
-    echo -ne "0" > ${DIRECTORY}/var/upgrade/upgrade_result
-    echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade finished successfully." >> ${DIRECTORY}/logs/upgrade.log
+    echo "$(date +"%Y/%m/%d %H:%M:%S") - Connected to manager." >> ${WAZUH_HOME}/logs/upgrade.log
+    echo -ne "0" > ${WAZUH_HOME}/var/upgrade/upgrade_result
+    echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade finished successfully." >> ${WAZUH_HOME}/logs/upgrade.log
 else
     # Restoring backup
-    echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade failed. Restoring..." >> ${DIRECTORY}/logs/upgrade.log
-    ${DIRECTORY}/bin/wazuh-control stop >> ${DIRECTORY}/logs/upgrade.log 2>&1
-    tar xzf ${DIRECTORY}/backup/backup_${VERSION}_[${BDATE}].tar.gz -C / >> ${DIRECTORY}/logs/upgrade.log 2>&1
-    echo -ne "2" > ${DIRECTORY}/var/upgrade/upgrade_result
-    ${DIRECTORY}/bin/wazuh-control start >> ${DIRECTORY}/logs/upgrade.log 2>&1
+    echo "$(date +"%Y/%m/%d %H:%M:%S") - Upgrade failed. Restoring..." >> ${WAZUH_HOME}/logs/upgrade.log
+    ${WAZUH_HOME}/bin/ossec-control stop >> ${WAZUH_HOME}/logs/upgrade.log 2>&1
```

Comment:
Why do you use `ossec-control` here and below? I would try to use `wazuh-control` and if it doesn't exist, I would use `ossec-control`.

Reply:
This was an error during the merge of the file. Nice catch! 
Will fix it.

Reply:
The ossec-control usage it's reverted now, it uses wazuh-control as before.
The need to check both, ossec-control or wazuh-control, we will handle it in a new issue as it is a legacy problem.
Thanks!